src/Microsoft.Extensions.Configuration.Abstractions/ConfigurationPath.cs

```diff
@@ -11,26 +11,44 @@ namespace Microsoft.Extensions.Configuration
     /// </summary>
     public static class ConfigurationPath
     {
+        /// <summary>
+        /// The deliminiter used for keys is ":".
```

Comment:
Sorry I missed this one. It has a typo in `deliminiter` and sounds a bit strange to me. I suggest: 

`The delimiter ":" used to separate individual keys in a path.`